extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/kafkastream/KafkaStreamsMetricsProducer.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.micrometer.runtime.binder.kafkastream;
```

Comment:
Again, torn here on the package. ;) Can we keep all things Kafka in one package? The class name is descriptive..